clang/include/clang/Lex/Preprocessor.h

```diff
@@ -2123,17 +2123,18 @@ class Preprocessor {
   char
   getSpellingOfSingleCharacterNumericConstant(const Token &Tok,
```

Comment:
Both the documentation and name of this function is incorrect now that the underlying functionality has changed - would you mind updating it to something more appropriate @Fznamznon? It used to return the character itself:
```
  /// Given a Token \p Tok that is a numeric constant with length 1,
  /// return the character.
```

But now returns the value of the character.

Reply:
I've updated the comment and the interface in 295e4f49aefb2b07501be9f845df598b3ee280f1

Reply:
Thank you @AaronBallman

Reply:
Thanks @AaronBallman!